probe/docker/controls.go

```diff
@@ -99,7 +99,7 @@ func (r *registry) execContainer(containerID string, req xfer.Request) xfer.Resp
 		AttachStdout: true,
 		AttachStderr: true,
 		Tty:          true,
-		Cmd:          []string{"/bin/sh"},
+		Cmd:          []string{"/usr/bin/env", "TERM=xterm", "/bin/sh", "-l"},
```

Comment:
or we could run `/bin/sh -c "TERM=xterm exec /bin/sh"`, which doesn't rely on env.